backend/src/openarchiefbeheer/destruction/api/filtersets.py

```diff
@@ -76,10 +78,28 @@ def filter_destruction_list_uuid(
 
 
 class DestructionListReviewItemFilterset(FilterSet):
```

Comment:
Would this not only solve the problem on the 'changes requested' view, but not on the 'normal' detail view of a destruction list?